### PR TITLE
chore: remove try in favour of ?

### PR DESCRIPTION
**Pull request by JosephGoulden**

### CODE REVIEW

**Comment by JosephGoulden** [comment]
[clabot:check]

-------- Original Message --------

File: serialization/src/impls.rs [hunk 1]
```diff
@@ -228,9 +228,9 @@ impl Serializable for Bytes {
 
 impl Deserializable for Bytes {
 	fn deserialize<T>(reader: &mut Reader<T>) -> Result<Self, Error> where T: io::Read {
-		let len = try!(reader.read::<CompactInteger>());
+		let len = reader.read::<CompactInteger>()?;
 		let mut bytes = Bytes::new_with_len(len.into());
-		try!(reader.read_slice(&mut bytes));
+		reader.read_slice(&mut bytes);
```
Comment (svyatonik) [inline serialization/src/impls.rs]: Please add `?`

File: keys/src/private.rs [hunk 2]
```diff
@@ -107,8 +107,8 @@ impl DisplayLayout for Private {
 
 impl fmt::Debug for Private {
 	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-		try!(writeln!(f, "network: {:?}", self.network));
-		try!(writeln!(f, "secret: {}", self.secret.to_hex::<String>()));
+		writeln!(f, "network: {:?}", self.network);
```
Comment (svyatonik) [inline keys/src/private.rs]: Please add `?`

File: keys/src/private.rs [hunk 3]
```diff
@@ -107,8 +107,8 @@ impl DisplayLayout for Private {
...
+		writeln!(f, "secret: {}", self.secret.to_hex::<String>());
```
Comment (svyatonik) [inline keys/src/private.rs]: Please add `?`

File: verification/src/accept_chain.rs [hunk 4]
```diff
@@ -65,9 +65,9 @@ impl<'a> ChainAcceptor<'a> {
 	}
 
 	pub fn check(&self) -> Result<(), Error> {
-		try!(self.block.check());
-		try!(self.header.check());
-		try!(self.check_transactions());
+		self.block.check();
```
Comment (svyatonik) [inline verification/src/accept_chain.rs]: Please add `?`

File: verification/src/accept_chain.rs [hunk 5]
```diff
@@ -65,9 +65,9 @@ impl<'a> ChainAcceptor<'a> {
...
+		self.header.check();
```
Comment (svyatonik) [inline verification/src/accept_chain.rs]: Please add `?`

File: verification/src/accept_chain.rs [hunk 6]
```diff
@@ -65,9 +65,9 @@ impl<'a> ChainAcceptor<'a> {
...
+		self.check_transactions();
```
Comment (svyatonik) [inline verification/src/accept_chain.rs]: Please add `?`

File: verification/src/accept_header.rs [hunk 7]
```diff
@@ -29,9 +29,9 @@ impl<'a> HeaderAcceptor<'a> {
 	}
 
 	pub fn check(&self) -> Result<(), Error> {
-		try!(self.version.check());
-		try!(self.work.check());
-		try!(self.median_timestamp.check());
+		self.version.check();
```
Comment (svyatonik) [inline verification/src/accept_header.rs]: Please add `?`

File: verification/src/accept_header.rs [hunk 8]
```diff
@@ -29,9 +29,9 @@ impl<'a> HeaderAcceptor<'a> {
...
+		self.work.check();
```
Comment (svyatonik) [inline verification/src/accept_header.rs]: Please add `?`

File: verification/src/accept_header.rs [hunk 9]
```diff
@@ -29,9 +29,9 @@ impl<'a> HeaderAcceptor<'a> {
...
+		self.median_timestamp.check();
```
Comment (svyatonik) [inline verification/src/accept_header.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 10]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
 		// Tx pool validation is not strinctly a matter of consensus.
-		try!(self.size.check());
-		try!(self.missing_inputs.check());
-		try!(self.maturity.check());
-		try!(self.overspent.check());
-		try!(self.sigops.check());
-		try!(self.double_spent.check());
-		try!(self.return_replay_protection.check());
-		try!(self.eval.check());
+		self.size.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 11]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
+		self.missing_inputs.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 12]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
+		self.maturity.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 13]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
+		self.overspent.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 14]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
+		self.sigops.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 15]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
+		self.double_spent.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 16]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
+		self.return_replay_protection.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

File: verification/src/accept_transaction.rs [hunk 17]
```diff
@@ -114,14 +114,14 @@ impl<'a> MemoryPoolTransactionAcceptor<'a> {
...
+		self.eval.check();
```
Comment (svyatonik) [inline verification/src/accept_transaction.rs]: Please add `?`

**Comment by JosephGoulden** [comment]
thanks @svyatonik, I've fixed those now.

**Pull request merged**